### PR TITLE
Fixing social login

### PR DESCRIPTION
**Pull request by Manjukcthapa**
# Description

Fixed the social login

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] There are no merge conflicts

### CODE REVIEW

File: config/keys.js [hunk 1]
```diff
@@ -7,11 +9,13 @@ module.exports = {
...
   googleAuth: {
     googleClientId: process.env.GOOGLE_CLIENTID,
-    googleClientSecret: process.env.GOOGLE_CLIENTSECRET
+    googleClientSecret: process.env.GOOGLE_CLIENTSECRET,
+    callbackURL: '/auth/google/callback' 
   },
   facebookAuth: {
-    facebookClientId: process.env.FACEBOOK_CLIENTID,
-    facebookClientSecret: process.env.FACEBOOK_CLIENTSECRET
+    facebookClientId: '482046309075252',
```
Comment (shweps13) [inline config/keys.js]: we cannot store keys in code - this is a gross violation
please, remove keys from here and add it to Heroku env!

make the same one operations on contest side

File: index.js [hunk 2]
```diff
@@ -1,42 +1,13 @@
 require("dotenv").config();
 checkConfig();
-const express = require("express");
-const app = express();
 const mongoose = require("mongoose");
-const cors = require("cors");
```
Comment (shweps13) [inline index.js]: I believe that we cannot remove basic libraries, like express or cors

File: config/keys.js (hunk 1 above)
Comment (Manjukcthapa) [reply config/keys.js]: I think Isla had stored the key in code. Okay, I will remove it.

File: index.js (hunk 2 above)
Comment (Manjukcthapa) [reply index.js]: Cors is removed here because it is in server.js. We don't need it twice.

Comment (shweps13) [reply index.js]: ok, got it

File: config/keys.js (hunk 1 above)
Comment (shweps13) [reply config/keys.js]: You definitely should get keys from process.env.somekey

Comment (shweps13) [reply config/keys.js]: don't forget that we have separate env for each server (master, deployment, contest)

**Pull request merged**